src/main/java/com/aca/armine/serialization/FileContentException.java

```diff
@@ -0,0 +1,11 @@
+package com.aca.armine.serialization;
+
+import java.util.logging.Logger;
+
+public class FileContentException extends RuntimeException {
+    public FileContentException(String message){
+        Logger logger = Logger.getGlobal();
```

Comment:
use Logger where you throw this exception, it will help you customize message, in addition here you don't use message parameter